### PR TITLE
Update API to return both package and repo information

### PR DESCRIPTION
**Pull request by samuelli**
Part of #531

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ Before using the API, please file an issue with your intention of using the API.
 
 You can access the API at `www.webcomponents.org/api/<method>`. Example search API call: https://www.webcomponents.org/api/search/input.
 
+## NPM packages
+For relevant endpoints, owner and repo can be replaced my the NPM's scope and package
```
Comment (aomarks) [inline API.md]: "replaced by the"

File: src/api.py [hunk 2]
```diff
@@ -130,6 +130,13 @@ def full_async(library_key, tag=None, brief=False, assume_latest=False):
       raise ndb.Return(None)
 
     result = {}
+    # Add NPM package fields
+    key = library_key.string_id()
+    if key.startswith('@'):
+      if key.split('/')[0] != '@@npm':
```
Comment (aomarks) [inline src/api.py]: double equals?

Comment (aomarks) [inline src/api.py]: nit: make a parts variable instead of splitting 3 times?

File: API.md (hunk 1 above)
Comment (samuelli) [inline API.md]: Done

File: src/api.py (hunk 2 above)
Comment (samuelli) [inline src/api.py]: Python not js

Comment (samuelli) [inline src/api.py]: Done.

**Pull request merged**